packages/common/source/utils/index.ts

```diff
@@ -1,5 +1,9 @@
 import mongoose from "mongoose";
+import sgMail from "@sendgrid/mail";
 
+const SENDGRID_API_KEY = process.env.SENDGRID_API_KEY;
+const INVITATION_JWT_SIGNATURE = process.env.INVITATION_JWT_SIGNATURE;
+sgMail.setApiKey(SENDGRID_API_KEY);
```

Comment:
Use newlines!

Reply:
You need to destructure whenever you write the variable name in two places like this:

const a = object.a;

becomes

const { a } = object;

Reply:
Ok